### PR TITLE
Adds a spoon

### PR DESCRIPTION
**Pull request by Moltijoe**
Adds a large spoon to hacked kitchen vendor

spoon warrior

also fixes the typo in the .dm file

:cl:  @cuackles @Moltijoe 
rscadd: Adds a spoon
imageadd: spoon
/:cl:

### CODE REVIEW

**Comment by Rdrazga** [comment]
Some might even call it comically large

File: yogstation.dme [hunk 1]
```diff
@@ -3492,7 +3492,7 @@
 #include "code\modules\vending\clothesmate.dm"
 #include "code\modules\vending\coffee.dm"
 #include "code\modules\vending\cola.dm"
-#include "code\modules\vending\drinnerware.dm"
```
Comment (ThatLing) [inline yogstation.dme:3495]: Undocumented change

Comment (MajManatee) [reply yogstation.dme:3495]: how dare he make it spelt right...

Comment (Moltijoe) [reply yogstation.dme:3495]: done

**Comment by cuackles** [comment]
@ToasterBiome / @Ebin-Halcyon must review pls

File: code/game/objects/items/twohanded.dm [hunk 2]
```diff
@@ -1235,3 +1235,31 @@
...
 		else
 			return TRUE
+
+/obj/item/twohanded/bigspoon
+	name = "big spoon"
+	desc = "For when you're only allowed one spoonful of something."
+	icon = 'yogstation/icons/obj/kitchen.dmi'
+	mob_overlay_icon = 'yogstation/icons/mob/clothing/back.dmi'
+	icon_state = "bigspoon"
+	item_state = "bigspoon0"
```
Comment (MajManatee) [inline code/game/objects/items/twohanded.dm:1245]: is this supposed to have a 0?

File: code/modules/vending/dinnerware.dm [hunk 3]
```diff
@@ -18,8 +18,10 @@
 					/obj/item/reagent_containers/glass/mixbowl = 3, // Yogs -- chef's mixing bowl 
 					/obj/item/reagent_containers/food/condiment/cinnamon = 5, // Yogs -- cinnamon shakers!
 					/obj/item/plate = 10) 
-	contraband = list(/obj/item/kitchen/knife/butcher = 2, // Yogs -- Pan
-					  /obj/item/melee/fryingpan = 2) // Yogs -- Pan
+	contraband = list(/obj/item/kitchen/knife/butcher = 2,
+					  /obj/item/melee/fryingpan = 2,	// Yogs -- Pan
+					  /obj/item/twohanded/bigspoon = 2, // Yogs -- Big spoon
```
Comment (MajManatee) [inline code/modules/vending/dinnerware.dm:23]:
```suggestion
					  /obj/item/twohanded/bigspoon = 1, // Yogs -- Big spoon
```
Yeah theres two chefs, but is two a good idea?

Comment (Moltijoe) [reply code/modules/vending/dinnerware.dm:23]: there's two of all the others

File: code/game/objects/items/twohanded.dm (hunk 2 above)
Comment (Moltijoe) [reply code/game/objects/items/twohanded.dm:1245]: yes, because it's the unwielded version, wielded version is given bigspoon1

**Review by MajManatee — APPROVED** [review]
shoulda made it make the pipe clatter sound effect

File: code/game/objects/items/twohanded.dm [hunk 4]
```diff
@@ -1235,3 +1235,31 @@
 			target.Paralyze(40)
 		else
 			return TRUE
+
+/obj/item/twohanded/bigspoon
+	name = "big spoon"
```
Comment (ReddicusDragon) [inline code/game/objects/items/twohanded.dm]:
```suggestion
	name = "comically large spoon"
```

**Review by Ebin-Halcyon — APPROVED** [review]
Yo bro can I have a sprite?
Only a 32x32 full.
*Pulls out comedically sized spoon*

**Comment by ofscarletdreams** [comment]
hello we are here from beyond the grave to repossess your silverware

**Pull request merged**